cnapy/octave_engine.py

```diff
@@ -1,6 +1,16 @@
 import tempfile
+import os
+import re
+from platform import system
+from pkg_resources import VersionConflict
 
 try:
+    if system() == "Linux":
+        octave_version = re.search(r'(?<=version).*?(?=\n|$|\.)',os.popen("octave -v").read())
+        octave_version = int(octave_version.group(0))
```

Comment:
Can there be cases that `octave_version` is None? Maybe when no octave is installed?